src/V3EmitCImp.cpp

```diff
@@ -887,19 +898,27 @@ void V3EmitC::emitcImp() {
     UINFO(2, __FUNCTION__ << ": " << endl);
     // Make parent module pointers available.
     const EmitCParentModule emitCParentModule;
+    std::list<std::deque<AstCFile*>> cfiles;
 
     // Process each module in turn
     for (const AstNode* nodep = v3Global.rootp()->modulesp(); nodep; nodep = nodep->nextp()) {
         if (VN_IS(nodep, Class)) continue;  // Imped with ClassPackage
         const AstNodeModule* const modp = VN_AS(nodep, NodeModule);
-        EmitCImp::main(modp, /* slow: */ true);
-        EmitCImp::main(modp, /* slow: */ false);
+        cfiles.emplace_back(std::deque<AstCFile*>{});
```

Comment:
`cfiles.emplace_back()` should work, no need to move construct from temporary with default constructor.